integration-tests/src/endpoints/feeds.py

```diff
@@ -0,0 +1,163 @@
+import numpy
+import pandas
+
+from endpoints.integration_tests import IntegrationTests
+
+
+class FeedsEndpointTests(IntegrationTests):
+    def __init__(self, file_path, access_token, url, progress):
+        super().__init__(file_path, access_token, url, progress=progress)
+
+    def test_feeds_id(self):
+        """Test retrieval of specific feed by ID"""
+        stable_id = "mdb-100"
+        response = self.get_response(f"v1/feeds/{stable_id}")
+        assert response.status_code == 200, (
+            f"Expected 200 status code for stable_id '{stable_id}', got "
+            f"{response.status_code}."
+        )
+        feed = response.json()
+        assert (
+            feed["id"] == stable_id
+        ), f"Expected feed ID '{stable_id}', got '{feed['id']}'."
+
+    def test_feeds_default(self):
+        """Test default feed retrieval"""
+        response = self.get_response("v1/feeds")
+        assert (
+            response.status_code == 200
+        ), f"Expected 200 status code, got {response.status_code}."
+        assert isinstance(response.json(), list), "Response should be a list."
+        assert len(response.json()) > 0, "Expected non-empty list of feeds."
+
+    def test_feeds_limit(self):
+        """Test feed retrieval with limit"""
+        response = self.get_response("v1/feeds", {"limit": 10})
+        assert (
+            response.status_code == 200
+        ), f"Expected 200 status code, got {response.status_code}."
+        feeds = response.json()
+        assert isinstance(feeds, list), "Response should be a list."
+        assert len(feeds) == 10, f"Expected 10 feeds, got {len(feeds)}."
+
+    def test_feeds_offset(self):
```

Comment:
Having disjoint lists returned only partially test this functionality. Have you considered something like having an offset of 1 and checking that only the first item of the first list is missing in the second list?
Or even simpler for the first retrieval set a limit of 2 and the second retrieval set an offset of 1 and a limit of 1. The sole item from the second list should be the same as the second item of the first list.

Reply:
Good point! I added `test_feeds_pagination_simple_offset_limit` and `test_feeds_pagination_incremental_offset`.

Reply:
Great. For me it was actually an OR, ie do one or the other, but the more the better, right?